tf2onnx/tfonnx.py

```diff
@@ -1633,6 +1633,11 @@ def logical_compare_op(ctx, node, name, args):
                 ctx.copy_shape(inp, inp_cast.output[0])
                 ctx.set_dtype(inp_cast.output[0], target_dtype)
 
+def logical_compareeq_op(ctx, node, name, args):
+    logical_compare_op(ctx, node, name, [])
+    ctx.insert_new_node_on_output("Not", node.output[0], name=utils.make_name(name),
```

Comment:
insert_new_node_on_output does not have shapes/dtypes params. so we have to set_shape/set_dtype after the insertion.

Reply:
this can fix some of the failed ut cases I guess.

Reply:
yeah, I was just debugging it :) Thought it was passing the kwargs through. Going to fix it.

Reply:
we could later add the params for it to keep it consistent with Graph.make_node().

Reply:
I think that fixed it.